components/picker/stories/picker.stories.js

```diff
@@ -56,39 +91,60 @@ export default {
 		isDisabled,
 		isLoading,
 		isInvalid,
-		content: { table: { disable: true } },
+		isHovered,
+		isActive,
```

Comment:
I saw there was a section about "read-only" pickers in the guidelines: https://spectrum.adobe.com/page/picker/#Read-only

Should we add that to the controls at all? It's not on the docs site, and the S2 picker Figma designs don't look like they have a read only option.

Reply:
There are no read-only styles in the CSS, so there isn't anything for us to show currently. It's unclear to me looking at that sparse guideline about whether anything is needed for follow-up work related to read-only; it looks like Spectrum Web Components displays read-only like the default and React Spectrum does not have this option.